packages/dev-server/index.js

```diff
@@ -19,19 +19,21 @@ const getPublic = (neutrino, options) => {
 
 
 module.exports = (neutrino, opts = {}) => {
+  const port = opts.port || 5000;
   const publicHost = getPublic(neutrino, opts);
   const host = getHost(publicHost);
+
   const options = merge.all([
     {
-      port: 5000,
+      port,
       https: false,
       contentBase: neutrino.options.source,
       open: false,
       hot: true,
       historyApiFallback: true,
       publicPath: '/',
       headers: {
-        host: publicHost
+        host: `${publicHost}:${port}`
```

Comment:
Is there a reason why we include it in the host if it's defined in `port`?

Reply:
This: https://github.com/mozilla-neutrino/neutrino-dev/pull/487/files#diff-e157c45007fb7ba6df1afdf15e78279eR46

I wouldn't work everywhere without doing it like this. Or maybe it should only be set for public?

Reply:
Maybe just `public`.

Reply:
I'll try it out and see what happens.

Reply:
My bad, I thought this was editing `devServer.host`, but this is actually the `HOST` header we are modifying. The host header needs the port as well, not just the domain/host portion.

Reply:
@edmorley could you give this a sanity check?